src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -176,52 +148,14 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
             {
               ...(vals as VenueInput),
               id: venueId,
-              userStatuses,
-              showUserStatus: showUserStatuses,
             },
             user
           );
-
-          //@debt Create separate function that updates the userStatuses separately by venue id.
-          if (
-            sovereignVenueId &&
-            sovereignVenue &&
-            sovereignVenueId !== venueId
-          )
-            await updateVenue(
-              {
-                id: sovereignVenueId,
-                name: sovereignVenue.name,
-                subtitle:
-                  sovereignVenue.config?.landingPageConfig.subtitle ?? "",
-                description:
-                  sovereignVenue.config?.landingPageConfig.description ?? "",
-                adultContent: sovereignVenue.adultContent ?? false,
-                profile_questions: sovereignVenue.profile_questions,
-                code_of_conduct_questions:
-                  sovereignVenue.code_of_conduct_questions,
-                userStatuses,
-                showUserStatus: showUserStatuses,
-                template: sovereignVenue.template,
-              },
-              user
-            ).then(() => {
-              if (sovereignVenue) {
-                dispatch(
-                  setSovereignVenue({
-                    ...sovereignVenue,
-                    userStatuses,
-                    showUserStatus: showUserStatuses,
-                  })
-                );
-              }
-            });
         } else
           await createVenue(
             {
               ...vals,
-              userStatuses,
-              showUserStatus: showUserStatuses,
+              parentId: "/playa",
```

Comment:
can we have it moved to constants somewhere? settings.ts for example

Reply:
fixed in 14efccebfb3354121951a37f42bc254dcad58ad5